pkgs/tools/networking/s3cmd/default.nix

```diff
@@ -1,11 +1,12 @@
 { stdenv, fetchurl, pythonPackages }:
 
 pythonPackages.buildPythonApplication rec {
-  name = "s3cmd-1.5.2";
+  name = "s3cmd-${version}";
+  version = "1.6.1";
   
   src = fetchurl {
```

Comment:
`fetchFromGitHub { owner = "s3tools"; repo = "s3cmd"; rev = "v${version}"; ... }` ought to work here